index.php

```diff
@@ -421,7 +421,7 @@
             </div>
         </section>
         <section class="cta">
-            <img src="images/icons/places/128/distributor-logo.svg">
+            <img src="images/icons/places/128/distributor-logo.svg" alt="Elementary OS logo">
```

Comment:
"elementary" should be written lowercase:
```suggestion
            <img src="images/icons/places/128/distributor-logo.svg" alt="elementary OS logo">
```